connector-packager/connector_packager/jar_jdk_packager.py

```diff
@@ -2,13 +2,74 @@
 import logging
 import subprocess
 import shutil
+import xml.etree.ElementTree as ET
 
 from pathlib import Path
 from .connector_file import ConnectorFile
 from .helper import check_jdk_environ_variable
+from .version import __min_version_tableau__
 
 JAR_EXECUTABLE_NAME = "jar.exe"
 logger = logging.getLogger(__name__)
+MANIFEST_FILE_TYPE = "manifest"
```

Comment:
reuse MANIFEST_FILE_TYPE for defining MANIFEST_FILE_NAME and MANIFEST_FILE_COPY_NAME